gcp_airflow_foundations/common/gcp/load_builder.py

```diff
@@ -54,11 +55,17 @@ def load_builder(
     ods_suffix = data_source.ods_suffix
     hds_suffix = data_source.hds_suffix
 
-    ods_table_config.table_id = f"{landing_zone_table_name_override}{ods_suffix}"
+    dag_table_id = table_config.table_name
+    ods_table_id = f"{landing_zone_table_name_override}{ods_suffix}"
+    hds_table_id = ""
+    if hds_table_config:
+        hds_table_id = f"{landing_zone_table_name_override}{hds_suffix}"
```

Comment:
I think hds_table_id is used only instead 'if hds_table_config'
should we move it inside the if statement, and get rid of 'hds_table_id = ""'